src/components/inputs/localized-multiline-text-input/translation-input.styles.js

```diff
@@ -0,0 +1,112 @@
+import { css } from '@emotion/core';
+import vars from '../../../../materials/custom-properties';
+
+/* we need this line-height to achieve 32px height when the component has only one row */
+const sizeInputLineHeight = '22px';
+
+// NOTE: order is important here
+// * a disabled-field currently does not display warning/error-states so it takes precedence
+// * a readonly-field cannot be changed, but it might be relevant for validation, so error and warning are checked first
+// how you can interact with the field is controlled separately by the props, this only influences visuals
+const getTextareaStyles = props => {
+  const baseInputStyles = [
```

Comment:
I know it doesn't make much of a difference but these could be hoisted on top of the function? Which would also imply we would invoke `css` once?